Resources/Prototypes/Catalog/Fills/Lockers/security.yml

```diff
@@ -42,25 +42,25 @@
     contents:
       - id: FlashlightSeclite
       - id: WeaponDisabler
-        prob: 0.3
+#        prob: 0.3 # DeltaV - just give them the damn disabler
       - id: ClothingBeltSecurityFilled
       - id: Flash
       - id: ClothingEyesGlassesSecurity
       - id: ClothingHeadsetAltSecurity
       - id: ClothingHandsGlovesCombat
       - id: ClothingShoesBootsJack
-      - id: ClothingOuterCoatWarden
-      - id: ClothingOuterWinterWarden
+#      - id: ClothingOuterCoatWarden # DeltaV - removed for incongruence
+#      - id: ClothingOuterWinterWarden # DeltaV - removed for incongruence
```

Comment:
Something something net negative of content + glacier.

Reply:
Ooooooo the big bad "net negative" + addressed and replied